spec/tli/tli_spec.rb

```diff
@@ -152,6 +152,23 @@
     end
   end
 
+  describe 'validate languages passed as parameters' do
+    context 'with valid languages' do
+      it 'should open the tli interactive mode' do
+        readline = double('readline')
+        @interactive_tli = Tli.new(readline)
+        expect(readline).to receive(:readline).with('> ', true)
+        @interactive_tli.invoke([])
+      end
+    end
+
+    context 'with invalid languages' do
+      it 'should raise error' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.